assign_rights/templates/rest_framework/api.html

```diff
@@ -0,0 +1,241 @@
+              </li>
+            {% endfor %}
+          </ul>
+        </div>
+      {% else %}
+        <a class="btn btn-primary js-tooltip nav-link" href="{{ request.get_full_path }}" rel="nofollow" title="Make a GET request on the {{ name }} resource">GET</a>
+      {% endif %}
+    </fieldset>
+  </form>
+{% endif %}
+
+{% if options_form %}
+  <form class="button-form" action="{{ request.get_full_path }}" data-method="OPTIONS">
+    <button class="btn btn-primary js-tooltip mb-2" title="Make an OPTIONS request on the {{ name }} resource">OPTIONS</button>
+  </form>
+{% endif %}
+
+{% if delete_form %}
+  <button class="btn btn-danger button-form js-tooltip" title="Make a DELETE request on the {{ name }} resource" data-toggle="modal" data-target="#deleteModal">DELETE</button>
+
+  <!-- Delete Modal -->
+  <div class="modal fade" id="deleteModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-body">
+          <h4 class="text-center">Are you sure you want to delete this {{ name }}?</h4>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+          <form class="button-form" action="{{ request.get_full_path }}" data-method="DELETE">
+            <button class="btn btn-danger">Delete</button>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+{% endif %}
+
+{% if filter_form %}
+  <button style="float: right; margin-right: 10px" data-toggle="modal" data-target="#filtersModal" class="btn btn-default">
+    <span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
+    {% trans "Filters" %}
+  </button>
+{% endif %}
+</div>
+
+  <div class="content-main" role="main"  aria-label="{% trans "main content" %}">
+    <div class="page-header">
+      <h1>{{ name }}</h1>
+    </div>
+    <div style="float:left">
+      {% block description %}
+        {{ description }}
+      {% endblock %}
+    </div>
+
+    {% if paginator %}
+      <nav style="float: right">
+        {% get_pagination_html paginator %}
+      </nav>
+    {% endif %}
+
+    <div class="request-info" style="clear: both" aria-label="{% trans "request info" %}">
+      <pre class="prettyprint"><b>{{ request.method }}</b> {{ request.get_full_path }}</pre>
+    </div>
+
+    <div class="response-info" aria-label="{% trans "response info" %}">
+      <pre class="prettyprint"><span class="meta nocode"><b>HTTP {{ response.status_code }} {{ response.status_text }}</b>{% autoescape off %}{% for key, val in response_headers|items %}
+<b>{{ key }}:</b> <span class="lit">{{ val|break_long_headers|urlize }}</span>{% endfor %}
```

Comment:
Not to nitpick, but how about using `<span class="font-weight-bold">` instead of `<b>` to make these bold for style instead of semantics?

Reply:
I've removed this block from the template and we're just using the base code provided by DRF for this block now.